src/pkgmt/config.py

```diff
@@ -1,6 +1,15 @@
 import toml
+import yaml
+from pathlib import Path
 
 
 def load():
-    with open('pyproject.toml') as f:
-        return toml.load(f)['tool']['pkgmt']
+    if Path('pyproject.toml').exists():
```

Comment:
we don't have documentation for this project so please create a `doc.md` and document this (that I'll look for a `pyproject.toml` or a `config.yaml`)